drivers/display/display_st7789v.c

```diff
@@ -363,6 +364,17 @@ static int st7789v_init(const struct device *dev)
 			return -EIO;
 		}
 	}
+	if (config->bl_ctrl_gpio.port != NULL) {
+		if (!device_is_ready(config->bl_ctrl_gpio.port)) {
+			LOG_ERR("Backlight GPIO device not ready");
+			return -ENODEV;
+		}
+
+		if (gpio_pin_configure_dt(&config->bl_ctrl_gpio, GPIO_OUTPUT_INACTIVE)) {
+			LOG_ERR("Couldn't configure Backlight pin");
+			return -EIO;
+		}
+	}
```

Comment:
-2, not like this and not in this PR.

Reply:
-ENOENT?

Reply:
Changes in [drivers/display/display_st7789v.c](https://github.com/zephyrproject-rtos/zephyr/pull/55577/commits/8fec1e03a69a7300acd1b736e0e726500b8d6c81#diff-73d589a4a3df091a005e026fb9c14c1ee3d878c8746f02cd4ee592aacc075e49)  are not correct and do not belong in this PR.

Reply:
Thanks, removed commits not related to this PR